examples/sample-config.json

```diff
@@ -21,7 +21,7 @@
     },
     "trainer" : {
         "val_check_interval" : 1, 
-        "max_steps" : 50,
+        "max_steps" : 10,
```

Comment:
not sure this change is needed

Reply:
This was done just to see if the code code run without mistakes